### PR TITLE
Added more specific info about initializing db.

### PR DESCRIPTION
**Pull request by iBobik**
Added .sql.gz to supported files and explanation to what database it will be imported.

### CODE REVIEW

File: mariadb/content.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ This is an optional variable. Set to `yes` to allow the container to be started
 
 # Initializing a fresh instance
 
-When a container is started for the first time, a new database `mysql` will be initialized with the provided configuration variables. Furthermore, it will execute files with extensions `.sh` and `.sql` that are found in `/docker-entrypoint-initdb.d`. You can easily populate your %%REPO%% services by [mounting a SQL dump into that directory](https://docs.docker.com/userguide/dockervolumes/#mount-a-host-file-as-a-data-volume) and provide [custom images](https://docs.docker.com/reference/builder/) with contributed data.
+When a container is started for the first time, a new database `mysql` will be initialized with the provided configuration variables. Furthermore, it will execute files with extensions `.sh`, `.sql` and `.sql.gz` that are found in `/docker-entrypoint-initdb.d`. You can easily populate your mariadb services by [mounting a SQL dump into that directory](https://docs.docker.com/userguide/dockervolumes/#mount-a-host-file-as-a-data-volume) and provide [custom images](https://docs.docker.com/reference/builder/) with contributed data. SQL files will be imported by default to database specified by the `MYSQL_DATABASE` variable.
```
Comment (tianon) [inline mariadb/content.md]: Sorry for the delay!  Just two minor comments:
1. "mariadb" (in "populate your mariadb services") should still be "%%REPO%%" ("populate your %%REPO%% services")
2. "by default to database specified" should be "by default to _the_ database specified"

Happy to take over the PR from here if you'd rather! :+1:

Comment (iBobik) [inline mariadb/content.md]: Thanks :-)

Comment (mozzymoz) [inline mariadb/content.md]: https://docs.docker.com/userguide/dockervolumes/#mount-a-host-file-as-a-data-volume does not exist

Comment (iBobik) [inline mariadb/content.md]: Seems they moved it without redirect. Committing.

**Pull request closed without merging**